### ISSUE TITLE
is recognized as a virus.

### ISSUE THREAD

**Issue by geepnozeex** [RubisetCie/god-mode#3]
is recognized as a virus.

**Comment by RubisetCie** [comment]
Well, I'm not surprised, and unfortunately I can't do anything... I only used documented Win32 API calls to create this program, and no specific exploit...

god-mode is a "root-kit" after all, so its technically correct, but a "root-kit" you control. :)

**Comment by geepnozeex** [comment]
for example, these analogues are not recognized as a virus
https://github.com/xbebhxx3/x3-tool/releases
https://github.com/mspaintmsi/superUser/releases

**Comment by RubisetCie** [comment]
Indeed, it's pretty odd... did you tried build it yourself from the source code ?

It may be due to the executable signature (my pre-built releases are unsigned), or it may be because I built them using MinGW rather than MSVC... (I know the generated executables differs)

However, I reiterate this is not really a problem on my end: since I don't know how the antivirus you're using makes detection. If there's a false positive, we should report it to the company behind it, or whitelist it in the meantime...
